CppTicTacToe/tictactoe.cpp

```diff
@@ -93,6 +95,7 @@ void display_info(){
 	// I could mess up the game_logic flow.
 }
 
+// Tony Comment
 void find_three_in_a_row(){
```

Comment:
Above I talked about making functions that do a single thing for us, and that take in data, perform a calculation (in a manner of speaking) and communicate back their findings. I want you to start the process of turning your code into code that does that here with this function. Make the `find_three_in_a_row` take in the array, search for a win condition, and return true or false if it finds that a player has won the game or not.